client/v2/client_test.go

```diff
@@ -257,16 +258,19 @@ func TestClient_Concurrent_Use(t *testing.T) {
 	wg.Add(3)
 	n := 1000
 
+	errC := make(chan error, 3)
```

Comment:
the `range` is executing concurrently with the `Wait`, and `t.Error` will allow the channel to drain